tutorials/applications/napari_imageJ.md

```diff
@@ -1,12 +1,14 @@
-# Napari + ImageJ How-to-Guide
+# napari + ImageJ how-to guide
 
 ImageJ is a Java-based image processing program that provides extensibility via Java plugins and recordable macros. It can display, edit, analyze, process, save, and print 8-bit color and grayscale, 16-bit integer, and 32-bit floating point images. It can read many image file formats, including TIFF, PNG, GIF, JPEG, BMP, DICOM, and FITS, as well as raw formats. It has a plethora of features that can be checked out [here](https://en.wikipedia.org/wiki/ImageJ#Features).
 
 People who wish to try their hands on ImageJ can do so by downloading and installing it using this [link](https://imagej.net/Fiji/Downloads)
 
-### Reading Image with ImageJ and Viewing them with Napari 
+### Reading image with ImageJ and viewing them with napari
```

Comment:
```suggestion
### Reading images with ImageJ and viewing them with napari
```

Reply:
Very minor, should be either "Reading images with ImageJ" or "Reading an image with ImageJ". Since the text below says "images", we'll use that for the title too.